### DISCUSSION TITLE
DAC board question regarding single SID mode

### DISCUSSION THREAD

**Discussion by thomassandberg** [frntc/SIDKick-pico#46] [General]
Quick question regarding the DAC board.

If I just want to utilize the DAC instead of the PWM for better sound quality - do I need to solder the A5/A8 to use the DAC in single SID mode?

**Comment by frntc** [comment]
You can use the DAC with a single SID (i.e. without connecting anything to A5/A8). Emulation and output are independent.

**Reply by thomassandberg** [reply]
Awesome. Thanks!
